Conformity/custom-rule/README.md

```diff
@@ -0,0 +1,51 @@
+# custom-rule
+
+Helps manage custom rules in C1 Conformity using 4 simple commands in the CLI:
+
+    * create
+    * get
+    * update
+    * delete
+
+Additionally, the list command retrieves all custom rules in an organization.
+
+Usage:
+_ "./custom-rule.py help" for overall help
+_ "./custom-rule.py command help" for help on the given command
+
+## conformity terraform provider
+
+The Conformity Terraform provider provides the ideal experience for automation scenarios using infrastructure as code:
+
+https://registry.terraform.io/providers/trendmicro/conformity/latest/docs
+
+# developing custom rules
+
+The custom rules framework allows creating rich rulesets for evaluating resource configuration. The framework documentation is the ideal resource for understanding rule logic and other settings at:
+
+https://cloudone.trendmicro.com/docs/conformity/in-preview-custom-rules-overview
+
+All interactions with the tool require arguments taken from the Conformity Dashboard, your Cloud console and YAML files. The files are stored in the /workspace folder.
+
+The following command will assist developing custom rules:
```

Comment:
More information on how to execute those and examples will be helpful

Reply:
Examples added in READ.ME including a sample dev session.